### PR TITLE
docs: Add minor changes for project build

### PR DESCRIPTION
**Pull request by zachaysan**
- [x] I have run [`pre-commit`](https://docs.flagsmith.com/platform/contributing#pre-commit) to check linting
- [x] I have filled in the "Changes" section below?
- [x] I have used a [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) title for this Pull Request

## Changes
Added a few small nice to haves that would have saved me some time.

## Other possible change
In the same file it says:
    You'll also need to ensure that you have a value for `POSTGRES_PASSWORD` set as an environment variable on your development machine.

But I have not needed to add this environment variable to get the application running or to get the tests to pass. Is this no longer needed?

### CODE REVIEW

**Review by dabeeeenster — APPROVED** [review]

**Comment by dabeeeenster** [comment]
> You'll also need to ensure that you have a value for POSTGRES_PASSWORD set as an environment variable on your development machine.

> But I have not needed to add this environment variable to get the application running or to get the tests to pass. Is this no longer needed?

Maybe you were just lucky and no PW was needed connecting to localhost?

**Comment by khvn26** [comment]
> But I have not needed to add this environment variable to get the application running or to get the tests to pass. Is this no longer needed?

That's correct! We plan on updating those docs very soon (https://github.com/Flagsmith/flagsmith/issues/2893)

File: docs/docs/deployment/hosting/locally-api.md [hunk 1]
```diff
@@ -436,9 +436,19 @@ You can run Flagsmith as a single application/docker container using our unified
...
+cd frontend
+npm install
+npm run bundledjango
+
+# Copy additional assets with Django
+cd ../api
+python manage.py collectstatic
+
+# Boot the server
+python manage.py runserver
```
Comment (khvn26) [inline docs/docs/deployment/hosting/locally-api.md:450]: We have Makefile targets for that. Can we use these?

Comment (matthewelwell) [reply docs/docs/deployment/hosting/locally-api.md:450]: @khvn26 interestingly, we _don't_ have a makefile target for `runserver`. I'm not sure if there was a reason though? 

I tend to agree that for these type of docs we _should_ use the makefile targets where possible but we should perhaps have a discussion (in the next eng meeting perhaps) about the makefiles now that we have onboarded a new engineer.

Comment (khvn26) [reply docs/docs/deployment/hosting/locally-api.md:450]: > interestingly, we don't have a makefile target for runserver

We do: `make serve` spins up gunicorn with livereload enabled.

Comment (khvn26) [reply docs/docs/deployment/hosting/locally-api.md:450]: If any points are to be made ahead of the discussion, I suggest adding them here: https://github.com/Flagsmith/flagsmith/issues/2893

Comment (matthewelwell) [reply docs/docs/deployment/hosting/locally-api.md:450]: > We do: `make serve` spins up gunicorn with livereload enabled.

I don't think this is the same though, right? `runserver` does other things (e.g. you don't have to set ALLOWED_HOSTS and DEBUG environment variables).

Comment (khvn26) [reply docs/docs/deployment/hosting/locally-api.md:450]: Good point! We probably should have those set up for the local environment regardless of which dev server we use.

**Review by matthewelwell — APPROVED** [review]

File: docs/docs/deployment/hosting/locally-api.md (hunk 1 above)
Comment (khvn26) [reply docs/docs/deployment/hosting/locally-api.md:450]: Resolving this in hopes of a later iteration — current docs are correct enough.

**Pull request merged**